src/test/util/random.h

```diff
@@ -27,6 +28,9 @@ enum class SeedRand {
 /** Seed the global RNG state for testing and log the seed value. This affects all randomness, except GetStrongRandBytes(). */
 void SeedRandomStateForTest(SeedRand seed);
 
+extern std::atomic<bool> seeded_g_prng_zero;
+extern std::atomic<bool> used_g_prng;
```

Comment:
nit: Try to adhere to symbol naming convention from *developer-notes.md*?
```suggestion
extern std::atomic<bool> g_seeded_prng_zero;
extern std::atomic<bool> g_used_prng;
```

Reply:
(Would prefer `g_seeded_global_prng_zero` in that case, even though I admit it is verbose).